task/log.html

```diff
@@ -108,7 +108,7 @@
         <script id="TestingOverview" type="text/template">
             <div id="testing">
                 {{total}} tests, {{passed}} passed, {{failed}} failed,
-                {{skipped}} skipped, {{left}} to go ({{retries}} retries).<br>
+                {{skipped}} skipped, {{left}} to go ({{retries}} failure and {{affected_retries}} affected retries).<br>
```

Comment:
That reads a bit weird -- shouldn't it be `{{retries}} failure retries` or just "retries"? Or perhaps "failures", but I'd only count "final" failures as such.

Reply:
ideas welcomed.  There is `Retry (be robust against unstable tests) and `Retry (test affected tests 3 times)`. There are rather different as one points to a problem, the other not. The first (here marked as `failure`) is also shown in the top list. The others are not, I just kept it as interesting number to see (as you can see whether no tests are affected or it ran hundred tests more then normally PR would). Do you suggest not showing this other number at all?

Reply:
How about `...(retries: N failures, M successes)`?
Or just `...(N failure retries, M success retries)`?

Reply:
I just meant that "(3 failure and 26 affected retries)" reads weird -- at least it should be failure*s*. Does it count "final" failures (did not succeed even after 3 tries) and "intermediate" failures (but retry succeeded) separately?

Reply:
The full line is:
```
N tests, M passed, K failed, L skipped, O to go (P retries of failures, R retries of successes)
```
- K is number of "final" failures (did not succeed even after 3 tries | failed while was affected)
- P is number of "intermediate" failures. But does not say anything about the whole test. It is just test that was `not ok` and also had `RETRY` (that is different from `test affected tests 3 times`)
- R is number of tests, that were `ok` but we retry them anyway because they were affected. If affected test fails, it is not retried at all.

We don't show if tests succeeded but had retry. But that can be easily seen from the overview as it lists tests - the ones with white background and retries (just the P ones) and with red background are failed tests that were not retried. So if there is `testName` with two white bakgrounds followed by one red one, it means it was retried twice and even the third run failed.

Reply:
I see you fixed the failure*s* now, thanks!